### PR TITLE
Add Cartools v1.1.3

### PR DESCRIPTION
**Pull request by Smeds**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/cartools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
```
Comment (fanninpm) [inline recipes/cartools/meta.yaml:22]: > `ERROR: recipes/cartools/meta.yaml:0: missing_tests: The recipe is missing tests.`
```suggestion

test:
  commands:
    - python CARTools.py --help

```

Comment (Smeds) [reply recipes/cartools/meta.yaml:29]: done

File: recipes/cartools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
[the remark this answers is not in the document]
Comment (Smeds) [reply recipes/cartools/meta.yaml:22]: done

File: recipes/cartools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{% set name = "cartools" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: eb9e939ae2819cb9020ac4a8c2ea7d3e983356f36d1b0bf4d2c02a3405d33f7f
```
[the remark this answers is not in the document]
Comment (Smeds) [reply recipes/cartools/meta.yaml]: done

**Comment by Smeds** [comment]
Any more updates needed?

**Comment by fanninpm** [comment]
@BiocondaBot please update

**Comment by fanninpm** [comment]
> Any more updates needed?

Are you satisfied with the artifacts?

**Comment by fanninpm** [comment]
@BiocondaBot please add label

File: recipes/cartools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 090875bed4491af3bf0a5e501ebd7fbc201272b3db4f736506fe63b9b202b848
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
```
Comment (dpryan79) [inline recipes/cartools/meta.yaml:15]: Please change this to `noarch: python` so conda knows to move things around during deployment.

**Review by dpryan79 — CHANGES_REQUESTED** [review]
One minor tweak needed

**Comment by corneliusroemer** [comment]
CI fails, so can't be merged

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Pull request merged**